pandas/util/_decorators.py

```diff
@@ -107,6 +108,16 @@ def _deprecate_kwarg(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             old_arg_value = kwargs.pop(old_arg_name, None)
+
+            if new_arg_name is None and old_arg_value is not None:
+                msg = (
+                    "the '{old_name}' keyword is deprecated and will be removed in a future version"
+                    "please takes steps to stop use of '{old_name}'"
+                ).format(old_name=old_arg_name)
```

Comment:
since we have this new capability for ``deprecate_kwarg`` if you wanted (in a followup PR), to change existing code to use this kwargs rather than the 'manual' way of warning (e.g. before we had this)

Reply:
Sure.  I'll take a look around and open a separate PR.

Reply:
@jreback following up.  I found that the 'manual' deprecation warnings in the repo target an entire function or class (and in some cases class attributes).  I didn't find any for keyword specific deprecations.   I suppose I can create an analogous decorator called ``deprecate_func()`` or ``deprecate_obj`` if that would be of value.